code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -20,6 +20,7 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/hud_icon = 'yogstation/icons/mob/antag_hud.dmi'
 	///Name of the antag hud we provide to this mob.
 	var/antag_hud_name
+	var/awake_stage = ANTAG_AWAKE //What stage we are of "waking up"
```

Comment:
Shouldn't this be on the Traitor datum if that's who we want to be able to wake up?

Reply:
Leaving on the generic antag datum for other antag awakenings.